content/blog/tdd-2025/index.Rmd

```diff
@@ -0,0 +1,66 @@
+---
+output: hugodown::hugo_document
+
+slug: tdd-2025
+title: Tidyverse developer day 2025
+date: 2025-07-16
+author: Max Kuhn
+description: >
+     Join us in Atlanta for tidyverse developer day on September 19, 2025!
+
+photo:
+  author: George Stagg
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [other] 
+tags: [tidyverse-dev-day]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+The 2025 Tidy Dev Day will be on September 19th! We'll gather after posit::conf near the conference venue (see details at the event page at the bottom) from 9am to 5pm.
```

Comment:
Now I have a new email telling me it ends at 4pm 🤣

Reply:
I'll take it out and they can look at the event docs.

Reply:
SINGLE SOURCE OF TRUTH for the win 🤣